src/algebra/ring/prod.lean

```diff
@@ -103,3 +104,24 @@ rfl
 end prod_map
 
 end ring_hom
+
+namespace ring_equiv
+variables (R S) [semiring R] [semiring S]
+
+/-- Swapping components as an equivalence of (semi)rings. -/
+def prod_comm : R × S ≃+* S × R :=
+{ map_add' := λ ⟨_, _⟩ ⟨_, _⟩, rfl, ..mul_equiv.prod_comm R S }
```

Comment:
I wonder, does
```suggestion
{ ..add_equiv.prod_comm R S, ..mul_equiv.prod_comm R S }
```
work here?

Reply:
It does, modulo the fact that `to_additive` was a bit too smart and turned `mul_equiv.prod_comm` into `add_equiv.sum_comm`.

Reply:
Sounds like you ned to tell it not to do that